pages/linux/toolbox.md

```diff
@@ -16,7 +16,7 @@
 
 `toolbox rmi {{image1 image2 ...}}`
 
-- Display help for a specific subcommand (such as `create`, `enter`, `rm`, `rmi`, etc.):
```

Comment:
Why this change? The subcommand exists.

Reply:
Yes, but the listing in this description is just to provide some examples, not to serve as an exhaustive enumeration of all subcommands, so IMO three examples should suffice.